### PR TITLE
Additional modules

### PR DESCRIPTION
**Pull request by dlech**
@laurensvalk did a much better job on the `umath` module in #63, but I did all of this already, so feel free to copy/steal/merge/whatever.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -5,8 +5,9 @@
     "python.linting.flake8Enabled": true,
     "python.linting.enabled": true,
     "python.languageServer": "Pylance",
-    "files.associations": {"*.inc": "restructuredtext"},
-    "restructuredtext.confPath": "${workspaceFolder}/doc/api",
+    "files.associations": {
```
Comment (dlech) [inline .vscode/settings.json:8]: This must have been an automatic change that sneaked in. Not intentional.

File: pyproject.toml [hunk 2]
```diff
@@ -13,7 +13,16 @@ classifiers = [
     "Development Status :: 3 - Alpha",
     "Programming Language :: Python :: Implementation :: MicroPython",
 ]
-include = ["pybricks/py.typed"]
+packages = [
```
Comment (dlech) [inline pyproject.toml:16]: This gives us code completion for all modules, not just the `pybricks` module.

**Comment by laurensvalk** [comment]
Nice! At a glance, it seems we haven't really done any duplicate work at all. 

We can probably start with this PR as-is and add the updated/cleaned apis to it. I'll have a look later.

**Comment by laurensvalk** [comment]
> We can probably start with this PR as-is and add the updated/cleaned apis to it. I'll have a look later.

I've had a look, and this looks like the way to go. I like the way you organized both the MicroPython libraries and the Pybricks package under `src`.

There's a few typing errors throughout (e.g. micropython.opt_level, stack_use), but we can address these when working through the modules in the same way I did `umath`.

We can keep this PR open until we've done a first pass (I'll volunteer) and then enable it in the IDE docs as well.

**Pull request merged**